### ISSUE TITLE
Output obj file from Manifold is broken

### ISSUE THREAD

**Issue by dqj5182** [hjwdzh/Manifold#16]
I am using Manifold for processing Acronym meshes to be watertight. However, many of the meshes get broken after processing with the watertight script. The sample obj file is in this Google Drive link and file name is mesh file name for Acronym. Could you look over the result and check why such problem occurs?

https://drive.google.com/file/d/1H3AUbm9nDgG3xhewmpuTNYm2xZ6xRFgq/view?usp=sharing

**Comment by eoozbq** [comment]
hello, I have same result, did you solved it?

**Comment by dqj5182** [comment]
As far as I believe, this problem cannot be solved within Manifold. I recommend using [mesh-fusion](https://github.com/davidstutz/mesh-fusion) if you strictly need watertight mesh.

**Comment by eoozbq** [comment]
"This seems like a great job. I noticed the issue you mentioned, which displays the same visual problem with the mesh (https://github.com/hjwdzh/ManifoldPlus/issues/15#issue-976452926), stating that it's caused by self-intersections, particularly in the 'car' category. It seems a bit different from the surface breakage you mentioned earlier, so I wonder if this problem is indeed due to self-intersections. I'm a beginner and don't know much about this. Do you have any recommended tutorials or videos where I can learn more about this?"

**Comment by dqj5182** [comment]
Yes. I think the problem of self-intersection with "car" category is similar with my problem. This is a mix of self-intersection and inverted mesh faces. I am not sure whether there is any tutorial regarding self-intersection problems. But, I think you would be more comfortable from starting with [occupancy networks code](https://github.com/autonomousvision/occupancy_networks) to be more familiar with mesh-fusion (occupancy networks use mesh-fusion).

**Comment by eoozbq** [comment]
thank you very much!!
